bin/ember-template-lint.js

```diff
@@ -157,7 +161,22 @@ function run() {
   for (let relativeFilePath of filesToLint) {
     let filePath = path.resolve(relativeFilePath);
     let fileName = relativeFilePath === STDIN ? filePathFromArgs : relativeFilePath;
-    let fileErrors = lintFile(linter, filePath, fileName.slice(0, -4));
+    let moduleId = fileName.slice(0, -4);
+    let fileErrors = lintFile(linter, filePath, moduleId);
+
+    if (printPending) {
+      let failingRules = fileErrors.reduce((memo, error) => {
+        if (memo.indexOf(error.rule) === -1) {
+          memo.push(error.rule);
+        }
+
+        return memo;
+      }, []);
+
+      if (failingRules.length > 0) {
+        filesWithErrors.push({ moduleId, only: failingRules });
```

Comment:
If we make `failingRules` a `Set`, this would end up being something like:

```suggestion
        filesWithErrors.push({ moduleId, only: Array.from(failingRules) });
```